src/php/server_fpm.rs

```diff
@@ -8,13 +8,14 @@ use crate::php::php_server::PhpServer;
 #[cfg(not(target_family = "windows"))]
 use crate::php::structs::PhpServerSapi;
 use std::process::Child;
+use crate::utils::network;
 
 // Possible values: alert, error, warning, notice, debug
 #[cfg(not(target_family = "windows"))]
 const FPM_DEFAULT_LOG_LEVEL: &str = "notice";
 
 #[cfg(not(target_family = "windows"))]
-const FPM_DEFAULT_PORT: u16 = 65535;
+const FPM_DEFAULT_PORT: u16 = 60000;
```

Comment:
Now this is just an arbitrary chosen value. 
Playing the with symfony binary with invalid `--port` I assume that they try 50 ports and then stop.
This would currently iterate above 5535 ports in the unlikely case that all are blocked.

Reply:
IIRC, Symfony is starting from 8000, but I don't know when it will stop. I definitely think we can let it go from 8000 to 65535 without risk. If it still can't find any port at 65535, it means that the current process doesn't have any permission to listen to a port, which is a different problem and I don't know how to solve it yet :)

Reply:
I will give it a test. Do you think it should always be a higher port than the one for the HTTP connection? 
Not that both (FPM and HTTP) find the same free port and collide upon start-up.

Reply:
60000 is perfect for now, I'm sure that it will take a lot of time before any issue could come up with this :)